wolfssl/wolfcrypt/settings.h

```diff
@@ -2592,12 +2592,22 @@ extern void uITRON4_free(void *p) ;
  * group */
 #ifdef HAVE_LIBOQS
 #define HAVE_PQC
+#define HAVE_FALCON
+#define HAVE_KYBER
 #endif
 
-#if defined(HAVE_PQC) && !defined(HAVE_LIBOQS)
-#error "You must have a post-quantum cryptography implementation to use PQC."
+#ifdef HAVE_PQM4
+#define HAVE_PQC
+#define HAVE_KYBER
+#endif
+
+#if defined(HAVE_PQC) && !defined(HAVE_LIBOQS) && !defined(HAVE_PQM4)
+#error Please do not define HAVE_PQC yourself.
```

Comment:
Seems odd to throw a `#error` on this. I could understand erroring if both `HAVE_LIBOQS` and `HAVE_PQM4` are set, but not for someone trying to define `HAVE_PQC` themselves.

Reply:
If they did it themselves the build would fail and they wouldn't understand why.  This tells them why.